app/views/logix/teachers.html.erb

```diff
@@ -75,7 +75,7 @@
         </p>
       <h3 class="card-title">Embed Interactive Circuits in PowerPoint</h4>
         <p class="card-text">
-          Make sure the project is public. Click on embed, to get the embed HTML5 code, then simply embed the circuit in your PowerPoint. You may need to use a powerpoint plugin like <a href="https://www.liveslides.com/" > Live Slides</a> to embed the live Circuit. The team is currently developing a custom plugin to make this process easier.
+          Make sure the project is public. Click on embed, get the embed HTML5 code, and embed the circuit in your PowerPoint. Use a powerpoint plugin like <a href="https://www.liveslides.com/" > Live Slides</a> to embed the live Circuit. Our team is currently developing a custom plugin to make this process easier.
```

Comment:
I don't think we need to change this somehow. What do you think @tachyons @ankitkataria ?  Rest seems fine.